source/getting-started/components/wazuh-indexer.rst

```diff
@@ -29,8 +29,9 @@ Wazuh uses four different indices to store different event types:
 | **wazuh** |--| **statistics**   | Stores data related to the :doc:`/getting-started/components/wazuh-server` performance. It is used by the web interface to represent the performance statistics.                                                                         |
 +---------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+ 
 
-.. thumbnail:: /images/getting-started/Wazuh-indexer.jpg
-   :title: Wazuh indexer cluster and alert document extract
+.. thumbnail:: /images/getting-started/wazuh-indexer.png
```

Comment:
This image is about indices stored in shards of a Wazuh indexer cluster

Reply:
I think the name "Wazuh indexer" is correct in the context of the image.